.github/CODEOWNERS

```diff
@@ -1 +1,4 @@
+* SaschaMann
 * @exercism/maintainers-admin
```

Comment:
```suggestion
```

You need to remove this line, else it will overriden your line (plus we don't need to be codeowners of everything if you are for now)